Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -15,7 +15,7 @@
     "version": {
         "Major": 2,
         "Minor": 170,
-        "Patch": 0
+        "Patch": 1
```

Comment:
@t-dedah please check if a package with this patch version already exists. If so, then you might have to bump up to a patch version which does not exist.
https://mseng.visualstudio.com/AzureDevOps/_packaging?_a=feed&feed=Codex-Deps

Reply:
Confirmed
https://mseng.visualstudio.com/AzureDevOps/_packaging?_a=package&feed=Codex-Deps&package=Mseng.MS.TF.DistributedTask.Tasks.AzureResourceGroupDeploymentV2&protocolType=NuGet&version=2.182.0&view=versions